ST_structs.h

```diff
@@ -1,333 +1,721 @@
-        max_bmass,    /* sum of mature biomass for all species in group */
-        killfreq,       /* kill group at this frequency: <1=prob, >1=# years */
-        ignition,       /* cheatgrass biomass (g/m2) that triggers potential ignition of a wildfire */
-        cheatgrass_coefficient,   /* intercept of the cheatgrass biomass-wildfire probability relationship */
-        wild_fire_slope,  /* slope of the cheatgrass biomass-wildfire probability relationship */	
-        xgrow,        /* ephemeral growth = mm extra ppt * xgrow */
-        slowrate,     /* user-defined growthrate that triggers mortality */
-        ppt_slope[3], /* res. space eqn: slope for wet/dry/norm yrs*/
-        ppt_intcpt[3],/* res. space eqn: intercept for "" ""*/
-		proportion_killed,      /* proportion killing  */
-		proportion_recovered,   /* proportion recovery after killing year */
-        proportion_grazing;     /* proportion grazing on grazing year */
+      /** \brief Number of yers resources can be stretched without killing the group. */
+  IntUS max_stretch,
+      /** \brief Max number of species that can add new plants per year*/
+        max_spp_estab,
+      /** \brief Number of species in the group*/
+        max_spp,
+      /** \brief Longest lifespan in resgroup. Used to malloc kills[] */
+        max_age,
+      /** \brief Don't start trying to grow until this year. */
+        startyr,
+      /** \brief Start year for kill frequency. 
+       * \sa killfreq */
+	      killfreq_startyr,
+      /** \brief Year in which group is extirpated. Setting to 0 turns off extirpation. */
+        extirp,
+      /** \brief Index number of this group in the RGroup array.
+       * \sa RGroup */
+        grp_num,
+      /** \brief SOILWAT2 vegetation type:
+       * STEPWAT2 inputs via "rgroup.in" are defined as: 1 for tree, 2 for shrub, 3 for grass, 4 for forb;
+       * however, the inputs get translated by get_SW2_veg_index() to SOILWAT2 values immediately upon reading the inputs. 
+       * \sa SW_Defines.h */
+        veg_prod_type,
+      /** \brief Start year for grazing frequency.
+       * \sa grazingfrq */
+        grazingfreq_startyr;
+      /** \brief Array of species indexes belonging to this group.
+       * \sa Species */
+  SppIndex *species;
+      /** \brief Input from table. */
+  RealF space,
+      /** \brief input space from table, rescaled if one or more rgroups is not established. */ 
+        min_res_req,
+      /** \brief density of mature plants in units of plants / m^2 */
+        max_per_sqm,
+      /** \brief Perform grazing on this group at this frequency. */
+		    grazingfrq,
+      /** \brief Number of mature plants allowed per plot. */
+        max_density,
+      /** \brief Max biomass of group. */
+        max_bmass,
+      /** \brief Kill group at this frequency.
+       * Values < 1 result in probablistic fires
+       * values > 1 result in fires ever killfreq years. 
```

Comment:
change to: 

values > 1 results in fires at a return interval = killfreq

Reply:
Addressed by commit 984bd13.